controllers/topolvm_controller.go

```diff
@@ -322,6 +323,28 @@ func getCsiResizerContainer() *corev1.Container {
 	return csiResizer
 }
 
+func getCsiSnapshoterContainer() *corev1.Container {
+
+	args := []string{
+		fmt.Sprintf("--csi-address=%s", DefaultCSISocket),
+		"--connection-timeout=15s",
```

Comment:
We dont have `connection-timeout` flag removed it from here and use the defaults?

Reply:
removed it.